.github/workflows/cmake.yml

```
@@ -771,6 +770,17 @@ jobs:
         path: ${{ matrix.codecov }}.xml
         retention-days: 1
 
+    - name: Test benchmarks (crashtest only, no coverage data collection)
+      if: contains(matrix.cmake-args, '-DWITH_BENCHMARKS=ON')
+      run: ctest --verbose -C Release -R ^benchmark_zlib$ --output-on-failure --max-width 120 -j ${{ matrix.parallels-jobs || '3' }}
+      working-directory: ${{ matrix.build-dir || '.' }}
+      env:
+        ASAN_OPTIONS: ${{ matrix.asan-options || 'verbosity=0' }}:abort_on_error=1:halt_on_error=1
+        MSAN_OPTIONS: ${{ matrix.msan-options || 'verbosity=0' }}:abort_on_error=1:halt_on_error=1
```

actionlint (1.7.4)

property "tsan-options" is not defined in object type {asan-options: string; build-config: string; build-dir: string; build-shared: string; build-src-dir: string; cflags: string; chost: string; cmake-args: string; codecov: string; compiler: string; cxx-compiler: string; cxxflags: string; gcov-exec: string; ldflags: string; msan-options: string; name: string; os: string; packages: string; parallels-jobs: number; readonly-project-dir: bool}

(expression)